.github/workflows/labeler.yml

```diff
@@ -53,12 +53,14 @@ jobs:
         env:
           event_json: "${{ toJSON(github.event) }}"
           GITHUB_TOKEN: ${{ steps.create_token.outputs.token }}
-        run:
-          ./venv/bin/python hacking/pr_labeler/label.py issue ${{ github.event.issue.number || inputs.number }}
+          number: "${{ github.event.issue.number || inputs.number }}"
```

Comment:
I wonder if we should consider removing `workflow_dispatch` from this workflow? The blog article mentions number as event data that users can't control. Providing an input in a workflow seems like a way to bypass that. Even though this touches on the whole "protecting us from ourselves" discussion.

Reply:
I use the `workflow_dispatch` to manually trigger dependency updates or to refresh a dependency update PR, so I'd prefer not to do this.